### PR TITLE
More verbose errors + improved python style

### PR DESCRIPTION
**Pull request by fansanelli**
Hello @frodrigo 
I've followed your suggestion and added an extra error and more sub-classes.
Please let me know what you think...
Bonus stage: improved the python style following (some) PEP8 suggestion.

### CODE REVIEW

File: plugins/TagFix_Vatin.py [hunk 1]
```diff
@@ -34,11 +35,12 @@ class TagFix_Vatin(Plugin):
 
     def init(self, logger):
         Plugin.init(self, logger)
-        self.errors[32601] = {"item": 3260, "level": 3, "tag": ["ref"], "desc": T_(u'Invalid value of tag "ref:vatin"') }
+        self.errors[99998] = {"item": 9998, "level": 3, "tag": ["ref", "fix:chair"], "desc": T_(u'Invalid format of tag "ref:vatin"')}
```
Comment (frodrigo) [inline plugins/TagFix_Vatin.py]: Invalid "value format", there allways someone to misunderstand....

File: plugins/TagFix_Vatin.py [hunk 2]
```diff
@@ -58,11 +60,15 @@ def it_vatin(self, vatin):
     def node(self, data, tags):
         if "ref:vatin" in tags:
             if tags["ref:vatin"].startswith("IT"):
-                if self.it_vatin(tags["ref:vatin"][2:]) == False:
-                    return {"class": 32601, "subclass": 1 }
+                if self.it_vatin(tags["ref:vatin"][2:]) is False:
+                    return {"class": 99998, "subclass": 1, "text": {"en": u"Invalid 'Partita IVA'"}}
```
Comment (frodrigo) [inline plugins/TagFix_Vatin.py]: Please use english text. Let's translators translate.

File: plugins/TagFix_Vatin.py (hunk 1 above)
Comment (frodrigo) [inline plugins/TagFix_Vatin.py]: Set class 32602 and item to 3260.

Comment (frodrigo) [reply plugins/TagFix_Vatin.py]:
```suggestion
        self.errors[32602] = {"item": 3260, "level": 3, "tag": ["ref", "fix:chair"], "desc": T_(u'Invalid format of tag "ref:vatin"')}
```

Comment (frodrigo) [reply plugins/TagFix_Vatin.py]:
```suggestion
        self.errors[32601] = {"item": 3260, "level": 3, "tag": ["ref", "fix:chair"], "desc": T_(u'Invalid value format of tag "ref:vatin"')}
```

File: plugins/TagFix_Vatin.py [hunk 3]
```diff
@@ -34,11 +35,12 @@ class TagFix_Vatin(Plugin):
 
     def init(self, logger):
         Plugin.init(self, logger)
-        self.errors[32601] = {"item": 3260, "level": 3, "tag": ["ref"], "desc": T_(u'Invalid value of tag "ref:vatin"') }
+        self.errors[99998] = {"item": 9998, "level": 3, "tag": ["ref", "fix:chair"], "desc": T_(u'Invalid format of tag "ref:vatin"')}
+        self.errors[99999] = {"item": 9999, "level": 3, "tag": ["ref", "fix:chair"], "desc": T_(u'Invalid value of tag "ref:vatin"')}
```
Comment (frodrigo) [inline plugins/TagFix_Vatin.py]:
```suggestion
```

File: plugins/TagFix_Vatin.py [hunk 4]
```diff
@@ -92,4 +98,3 @@ def test(self):
         assert a.node(None, {"ref:vatin": "ITAAAAAAAAAAA"})
         # missing country code
         assert a.node(None, {"ref:vatin": "11111111115"})
-
```
Comment (frodrigo) [inline plugins/TagFix_Vatin.py]: Please, keep a blank line at the end.

Comment (fansanelli) [reply plugins/TagFix_Vatin.py]: restored.. anyway it is a PEP8 standard, but I'm for the blank line too

File: plugins/TagFix_Vatin.py (hunk 2 above)
Comment (fansanelli) [reply plugins/TagFix_Vatin.py]: fixed, of course is better to only on text for all future countries

**Comment by frodrigo** [comment]
Look good thank you.

**Pull request merged**